README.md

```diff
@@ -15,6 +15,37 @@ Each player object contains direction and button actions. These are updated by t
 . Check for gamepad button presses (i.e. ‘justDown()’ functionality for gamepads)  
 . Check the last device type used for interaction.  
 
+## Installation
+
+```
+npm install phaser3-merged-input
+```
+
+Then add the plugin to Phaser 3's configuration:
```

Comment:
I would make clear that this is the `global` phaser conf. you could alternatively add this as scene plugin using `this..load.scenePlugin("mergedInput");`

Reply:
Fixed that, thank you. I am new to Phaser's plugins.